Rakefile

```diff
@@ -25,6 +25,36 @@ task(:push) do
   sh("bundle exec fastlane release")
 end
 
+task(:generate_team_table) do
+  require 'json'
+  content = ["<table>"]
+
+  contributors = JSON.parse(File.read("team.json"))
+  counter = 0
+  number_of_rows = 5
+
+  contributors.keys.shuffle.each do |github_user|
+    user_content = contributors[github_user]
+
+    content << "<tr>" if counter % number_of_rows == 0
+    content << "<td>"
+    content << "<img src='https://github.com/#{github_user}.png?size=200' width=140>"
```

Comment:
Why not use https://github.com/lacostej.png?size=140 and drop the `width` attribute?